app/views/partials/_favorite_team_game.html.erb

```diff
@@ -0,0 +1,136 @@
+            </div>
+          </div>
+        <% end %>
+        <div class="team__spacer"></div>
+        <div class="team">
+          <div class="team__text-container team__text-container--home">
+            <div class="team__text team__text--name">
+              <%= game.home_team.name %>
+            </div>
+            <div class="team__text team__text--score">
+            </div>
+          </div>
+          <div class="team__logo">
+            <%= game.home_team.logo_url %>
+          </div>
+        </div>
+      <% elsif game.user_pick_team(current_user, true) == game.home_team %>
+        <div class="team">
+          <div class="team__logo">
+            <%= game.away_team.logo_url %>
+          </div>
+          <div class="team__text-container team__text-container--away">
+            <div class="team__text team__text--name">
+              <%= game.away_team.name %>
+            </div>
+            <div class="team__text team__text--score">
+            </div>
+          </div>
+        </div>
+        <div class="team__spacer"></div>
+        <%= link_to pick_path(game.user_pick(current_user, true)), method: "delete", class: "team team--picked" do %>
+          <div class="team__text-container team__text-container--home">
+            <div class="team__text team__text--name">
+              <%= game.home_team.name %>
+            </div>
+            <div class="team__text team__text--score">
+            </div>
+          </div>
+          <div class="team__logo">
+            <%= game.home_team.logo_url %>
+          </div>
+        <% end %>
+      <% end %>
+    <% else %>
+      <%= link_to picks_path(game_id: game.id, picked_team_id: game.away_team_id), method: "post", class: "team" do %>
+        <div class="team__logo">
+          <%= game.away_team.logo_url %>
+        </div>
+        <div class="team__text-container team__text-container--away">
+          <div class="team__text team__text--name">
+            <%= game.away_team.name %>
+          </div>
+          <div class="team__text team__text--record">
+            <% if game.away_team.wins %>
+              <%= game.away_team.wins %>
+            <% else %>
+              0
+            <% end %>
+            <% if game.away_team.losses %>
+              - <%= game.away_team.losses %>
+            <% else %>
+              - 0
+            <% end %>
+          </div>
+        </div>
+      <% end %>
+      <div class="team__spacer"></div>
+      <%= link_to picks_path(game_id: game.id, picked_team_id: game.home_team_id), method: "post", class: "team" do %>
+        <div class="team__text-container team__text-container--home">
+          <div class="team__text team__text--name">
+            <%= game.home_team.name %>
+          </div>
+          <div class="team__text team__text--record">
+            <% if game.home_team.wins %>
+              <%= game.home_team.wins %>
+            <% else %>
+              0
+            <% end %>
```

Comment:
we can avoid this nonsense by changing the default in the schema to 0.

Reply:
Oh, that would be amazing. I didn't even consider that. Will that require a migration?

Reply:
Yeah. No biggie, I just didn't want to cram a bunch more into this pr. Plus I've got the haml version teed up, so any edits to templates probably oughta wait for that.